### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by wackxu**
**What this PR does / why we need it**:

 The file is just a simple copy of pkg/api/testapi.go to remove the dependeny to testapi, so we
 do not need add the test file. We should avoid duplicating code as much as we can.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

#50741 I have open this pr to fix the TODO in testapi_test.go : add some test case that have groups.
It needs approve, thanks.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wackxu** [comment]
@davidopp @jayunit100 Please take a look at this PR, it needs your approve, thanks.

**Comment by wackxu** [comment]
@davidopp @jayunit100 Please take a look at this PR, it needs your approve, thanks.

File: plugin/pkg/scheduler/util/testutil.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ import (
 	_ "k8s.io/kubernetes/pkg/api/install"
 )
 
+// The file is just a simple copy of pkg/api/testapi.go to remove the dependeny to testapi, so we
```
Comment (dims) [inline plugin/pkg/scheduler/util/testutil.go]: Do we have to keep this file and testapi.go in sync?

**Comment by dims** [comment]
@wackxu let me close this for now, please reopen if needed

/close

**Pull request closed without merging**